### PR TITLE
Adding option to specify advanced flags

### PR DESCRIPTION
**Pull request by jdewinne**
Allows specifying advanced flags in testgrid.
https://kurl.sh/docs/install-with-kurl/advanced-options

### CODE REVIEW

**Comment by jdewinne** [comment]
This is needed in order to be able to write an automated test for the labels flag
https://github.com/replicatedhq/kURL/pull/2445

File: testgrid/web/src/components/views/InstanceTable.jsx [hunk 1]
```diff
@@ -278,6 +296,12 @@ export default class InstanceTable extends React.Component {
         <tr key={kurlUrl}>
           <td>
             <span className="url" onClick={() => this.viewInstanceInstaller(this.props.instancesMap[kurlUrl][0])}>{kurlUrl}</span>
+            {this.props.instancesMap[kurlUrl][0].kurlFlags &&
+              <div>
+                <span>{' -> '}</span>
+                <span className="url" onClick={() => this.viewFlags(this.props.instancesMap[kurlUrl][0])}>{this.props.instancesMap[kurlUrl][0].kurlFlags}</span>
+              </div>
+              }
```
Comment (DanStough) [inline testgrid/web/src/components/views/InstanceTable.jsx:304]: Do you think there is an advantage for opening the modal here? 

```suggestion
            {this.props.instancesMap[kurlUrl][0].kurlFlags &&
              <span> [{this.props.instancesMap[kurlUrl][0].kurlFlags}] </span>
              }
```

You need to remove the state change if you opt for this.

**Review by DanStough — CHANGES_REQUESTED** [review]
I think the modal is overkill but feel free to disagree.

**Comment by emosbaugh** [comment]
Looks like this was merged in https://github.com/replicatedhq/kURL/pull/2736. Closing

**Pull request closed without merging**